addon/running-transition.js

```diff
@@ -34,6 +34,11 @@ export default class RunningTransition {
 
   _invokeAnimation() {
     return this.animation.run(this.animationContext).then(() => {
+      this.animationContext.oldElement = null;
+      this.animationContext.newElement = null;
+      this.animationContext.older.forEach((entry) => {
+          entry.element = null;
+      });
```

Comment:
I'm new to this lib code, so I investigated leakage points and removed lekage sources one by one using my chrome dev tools till there was no more node retainers, and I agree that some nulling might be overkill, but it worked!